bindings/java/iota-wallet-java/src/main/java/org/iota/types/account_methods/DestroyAlias.java

```diff
@@ -0,0 +1,23 @@
+package org.iota.types.account_methods;
+
+import org.iota.types.TransactionOptions;
+import org.iota.types.ids.AliasId;
+
+/// Destroy an alias output. Outputs controlled by it will be sweeped before if they don't have a
+/// storage deposit return, timelock or expiration unlock condition. The amount and possible native tokens will be
+/// sent to the governor address.
```

Comment:
```suggestion
/// Destroy an alias output.
```
Sweeping will not automatically happen with https://github.com/iotaledger/wallet.rs/pull/1440

Reply:
Can I just commit your comment suggestion or need I change anything?

Reply:
Should be good